corehq/apps/app_manager/templates/app_manager/form_view.html

```diff
@@ -181,8 +181,8 @@
               <div id="warningContent" class="collapse">
                 <p>
                   {% blocktrans with type=case_property_warning.type count=case_property_warning.count limit=case_property_warning.limit %}
-                    The '{{ type }}' case has {{ count }} properties. We
-                    recommend keeping case properties below {{ limit }}, otherwise
+                    The '{{ type }}' case has a total of {{ count }} custom properties. We
+                    recommend at most {{ limit }} custom properties per case type, otherwise
                     you may run into performance issues at the time of data
                     collection and analysis.
```

Comment:
Should this be kept in sync with this other similar message?

https://github.com/dimagi/commcare-hq/blob/0a27457d9c285f02e64c5cb283ebd000a857fa23/corehq/apps/data_dictionary/static/data_dictionary/js/data_dictionary.js#L620-L622